docs/index.rst

```diff
@@ -19,6 +19,7 @@ Utility for ingesting large survey data into HiPSCat structure.
    catalogs/allwise
    catalogs/neowise
    catalogs/tic
+   Notebooks <notebooks>
```

Comment:
Looks like updating the copier template plopped this down randomly into the catalogs list